openff/interchange/_tests/interoperability_tests/test_openmm.py

```diff
@@ -429,6 +446,19 @@ def test_combine_compatibility(self, de_force_field):
 
         assert isinstance(exception.value.__cause__, AssertionError)
 
+    def test_nocutoff_when_nonperiodic(self, de_force_field):
+        system = Interchange.from_smirnoff(
+            de_force_field,
+            MoleculeWithConformer.from_smiles("CCO").to_topology(),
+        ).to_openmm(combine_nonbonded_forces=False)
+
+        for force in system.getForces():
+            if type(force) in (
+                openmm.NonbondedForce,
+                openmm.CustomNonbondedForce,
+            ):
+                assert force.getNonbondedMethod() == openmm.NonbondedForce.NoCutoff
+
```

Comment:
There's a lot going on here, but this is the key behavior I recall you wanting @jthorton  - anything else I should be privy to?

Reply:
Thanks @mattwthompson this is fantastic and is what I need!